docs/quantized_ops.md

```diff
@@ -1,4 +1,4 @@
-Quantized Operations for XLA device (Experimental feature)
+# Quantized Operations for XLA device (Experimental feature)
 --------------------------
```

Comment:
I felt like it would be helpful to provide a runnable example in https://github.com/pytorch/xla/tree/master/examples for people to get a felling of how to use it. Can you add this to your roadmap and maybe we can do it next do cifx it?

Reply:
Sure, let's add an example. But also want to note that I already had runnable example in the existing doc.